src/Akeneo/Pim/Structure/Bundle/Resources/workspaces/settings-ui/src/pages/AttributeGroupsIndex.tsx

```diff
@@ -39,10 +64,59 @@ const AttributeGroupsIndex: FC = () => {
           {translate('pim_enrich.entity.attribute_group.result_count', {count: groupCount.toString()}, groupCount)}
         </PageHeader.Title>
       </PageHeader>
-      <PageContent>
-        <AttributeGroupsDataGrid groups={groups} onGroupCountChange={setGroupCount} />
-      </PageContent>
-    </>
+      <Content>
+        <AttributeGroupsDataGrid
+          attributeGroups={attributeGroups}
+          onGroupCountChange={setGroupCount}
+          isItemSelected={isItemSelected}
+          selectionState={selectionState}
+          onSelectionChange={onSelectionChange}
+          selectedCount={selectedCount}
+          onSelectAllChange={onSelectAllChange}
+        />
+      </Content>
+      {!isPending && (
+        <Toolbar isVisible={!!selectionState}>
+          <Toolbar.SelectionContainer>
+            <Checkbox checked={selectionState} onChange={value => onSelectAllChange(value)} />
+            <Dropdown>
+              <ArrowDownIcon onClick={openDropdown} />
+              {isDropdownOpen && (
+                <Dropdown.Overlay onClose={closeDropdown}>
+                  <Dropdown.Header>
+                    <Dropdown.Title>Select</Dropdown.Title>
+                  </Dropdown.Header>
+                  <Dropdown.ItemCollection>
+                    <Dropdown.Item
+                      onClick={() => {
+                        onSelectAllChange(true);
+                        closeDropdown();
+                      }}
+                    >
+                      All Attribute groups
+                    </Dropdown.Item>
+                    <Dropdown.Item
+                      onClick={() => {
+                        onSelectAllChange(false);
+                        closeDropdown();
+                      }}
+                    >
+                      No Attribute groups
+                    </Dropdown.Item>
+                  </Dropdown.ItemCollection>
+                </Dropdown.Overlay>
+              )}
+            </Dropdown>
+          </Toolbar.SelectionContainer>
+          <Toolbar.LabelContainer>
+            {translate('pim_enrich.entity.attribute_group.selected', {count: selectedCount}, selectedCount)}
+          </Toolbar.LabelContainer>
+          <Toolbar.ActionsContainer>
+            <ModalMassDeleteAttributeGroups attributeGroups={selection.collection} onConfirm={() => {}} />
```

Comment:
`selection.collection` will be empty if the use click on select all. We need to give the whole attribute groups in this case in order to have the right number displayed

```suggestion
            <ModalMassDeleteAttributeGroups attributeGroups={selectionState === true ? selection.collection : attributeGroups} onConfirm={() => {}} />
```

Reply:
Fixed that in RAB-1294